### PR TITLE
docs: update Flutter Web Example 

### PR DESCRIPTION
**Pull request by sashatalalasha**
## Proposed changes
This PR installs new versions of dependencies, fixes Ory base URL used for Ory Proxy and updates the guide. 

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Comment by Benehiko** [comment]
Don't use the ory proxy for this please! I intentionally removed the proxy usage with the flutter example due to the test flakes. The proxy makes the flutter app significantly slower to load in the browser since it pushes all the flutter assets through the proxy! with the tunnel it's only serving Ory-related data and the flutter app is loaded directly from its own dev server.

File: code-examples/protect-page-login/flutter_web_redirect/env [hunk 1]
```diff
@@ -1 +1 @@
-ORY_BASE_URL=http://localhost:3005
+ORY_BASE_URL=http://localhost:3005/.ory
```
Comment (Benehiko) [inline code-examples/protect-page-login/flutter_web_redirect/env:1]: revert please!

**Review by Benehiko — CHANGES_REQUESTED** [review]
As I have commented on the PR - we shouldn't use the Ory Proxy with the flutter example!

File: code-examples/protect-page-login/flutter_web_redirect/env (hunk 1 above)
Comment (sashatalalasha) [reply code-examples/protect-page-login/flutter_web_redirect/env:1]: ok thanks. I'll also update the documentation as it says to use Ory Proxy.

Comment (Benehiko) [reply code-examples/protect-page-login/flutter_web_redirect/env:1]: Thank you :)

**Review by Benehiko — APPROVED** [review]
LGTM :)

**Pull request merged**